crucible-llvm/src/Lang/Crucible/LLVM/Intrinsics/Libc.hs

```diff
@@ -870,6 +894,18 @@ callFloor ::
   OverrideSim p sym ext r args ret (RegValue sym (FloatType fi))
 callFloor bak (regValue -> x) = liftIO $ iFloatRound @_ @fi (backendGetSym bak) RTN x
 
+-- | An implementation of @libc@'s @fma@ function.
+callFMA ::
+     forall fi p sym bak ext r args ret
+   . IsSymBackend sym bak
+  => bak
+  -> RegEntry sym (FloatType fi)
+  -> RegEntry sym (FloatType fi)
+  -> RegEntry sym (FloatType fi)
+  -> OverrideSim p sym ext r args ret (RegValue sym (FloatType fi))
+callFMA bak (regValue -> x) (regValue -> y) (regValue -> z) = liftIO $
+  iFloatFMA @_ @fi (backendGetSym bak) RNE x y z
```

Comment:
How do we know that `RNE` is the right mode here?

Reply:
IEEE 754 has [this](https://iremi.univ-reunion.fr/IMG/pdf/ieee-754-2008.pdf) to say:

> The roundTiesToEven rounding-direction attribute shall be the default rounding-direction attribute for
results in binary formats. The default rounding-direction attribute for results in decimal formats is language-
defined, but should be roundTiesToEven.

So for most use cases, `RNE` (Round to Nearest Even) will replicate the behavior you'd see most of the time in `libc`. That being said, it would be worth documenting this somewhere, since this is not even the only function where `RNE` is used without explanation. I'll push a separate commit to do that.

We should also make a note somewhere that `crucible-llvm` doesn't allow changing the rounding mode, which is the only scenario (AFAICT) where this behavior would ever differ.